### PR TITLE
Add link to new api documentation

### PR DESCRIPTION
**Pull request by David-Wobrock**
https://botify.atlassian.net/browse/DATA-12310
In order to move to the new API documentation, we want to add a big red link to the old API doc's that redirects to the new one

image

On hold while we didn't update the DNS yet.

### CODE REVIEW

File: api_docs/templates/analysis_datamodel.html [hunk 1]
```diff
@@ -35,7 +35,7 @@ <h2>Analysis Datamodel</h2>
     <button type="submit" class="btn btn-default">OK</button>
     <span id="loading-message-token">Loading...</span>
     <span id="error-message-token" class="alert alert-danger"></span>
-    <div><a href="https://developers.botify.com/api/authentication/" class="link-token">Where can I find my token ?</a></div>
+    <div><a href="/api/authentication/" class="link-token">Where can I find my token ?</a></div>
```
Comment (SofienBenAyed) [inline api_docs/templates/analysis_datamodel.html:38]:
```suggestion
    <div><a href="/api/authentication/" class="link-token">Where can I find my token?</a></div>
```
By the way :)

**Review by y01s — APPROVED** [review]
LGTM
One small question: Did the design team approve the usage of this red? It doesn't really look good :p

**Review by SofienBenAyed — COMMENTED** [review]
It looks good! :+1:
About the red color on the purple background, the contrast is not high enough: https://webaim.org/resources/contrastchecker/?fcolor=CC0000&bcolor=5E4C83. Maybe we could change the color?

**Comment by David-Wobrock** [comment]
Sure we can change the colour. What would you suggest? :)
As a reminder, it's the colour of a link on a website we are deprecating.

**Comment by y01s** [comment]
> Sure we can change the colour. What would you suggest? :)
> As a reminder, it's the colour of a link on a website we are deprecating.

The link that Sofien sent is just brilliant. This color they suggested looks good #FF8585

**Review by y01s — APPROVED** [review]

**Comment by David-Wobrock** [comment]
Thanks for the help :hugs:

**Review by SofienBenAyed — APPROVED** [review]
Nice! :+1:

**Comment by David-Wobrock** [comment]
I'll handle the merge once the project is ready :)

**Pull request merged**